pkg/util/slices.go

```diff
@@ -1,3 +1,6 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
```

Comment:
why didn't the license check action work?

Reply:
https://github.com/cloudoperators/greenhouse/actions/runs/8823560471/job/24224268387